### PR TITLE
ENG-1851 - feat: add card example

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

Bootstrap project with card example

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "basis-theory-js-examples",
+  "version": "1.0.0",
+  "description": "Basis Theory Vanilla JS examples",
+  "main": "index.js",
+  "repository": "https://github.com/Basis-Theory/basis-theory-js-examples",
+  "license": "MIT",
```
Comment (brigonzalez) [inline package.json:7]:
```suggestion
  "license": "Apache-2.0",
```

File: card/public/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<body>
+<div class="container">
+  <div id="form">
+    <div class="row" id="success">
+      Test card charged $10.99 with Stripe.
+    </div>
+    <div class="row" id="error">Error occurred - check the console</div>
+    <div class="row row-input" id="card"></div>
+    <div class="row row-helper">
+      Give "4242 4242 4242 4242" with any expiration and CVC a try.
```
Comment (brigonzalez) [inline card/public/index.html:27]:
```suggestion
      Try using "4242 4242 4242 4242" with any expiration and CVC.
```

File: cypress/integration/card.spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  cvc: 1234,
+};
+
+context('card tokenization', () => {
+  beforeEach(() => {
+    cy.visit('card');
+    cy.wait(1500);
+  });
+
+  [visa, amex].forEach(({ brand, cardNumber, cvc }) =>
```
Comment (brigonzalez) [inline cypress/integration/card.spec.js:22]: 💯

**Review by brigonzalez — CHANGES_REQUESTED** [review]
🙌 small suggestions

File: cypress/integration/card.spec.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    it(`should tokenize/charge ${brand} card`, () => {
```
[the remark this answers is not in the document]
Comment (djejaquino) [reply cypress/integration/card.spec.js:23]: Maybe. Got any ideas?

**Review by lcschy — APPROVED** [review]

**Pull request merged**